src/Land/Model/source.jl

```diff
@@ -1,4 +1,30 @@
 #### Land sources
+export PhaseChange
+
+function heaviside(x::FT) where {FT}
+    if x > FT(0)
+        output = FT(1)
+    else
+        output = FT(0)
+    end
+    return output
+end
+
+
+abstract type Source{FT <: AbstractFloat} end
```

Comment:
I think this will conflict with `Source`, now defined in `BalanceLaws`-- feel free to change to `LandSource`. Or, if you're feeling adventurous, you can try out the new source specification :)

Reply:
changed to SoilSource for now. I need to look through this again to figure out what to add https://github.com/CliMA/ClimateMachine.jl/pull/1634/files

Reply:
gonna punt on this until we have more sources!